src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -1050,6 +1054,8 @@ impl LinkCollector<'_, '_> {
             };
         let mut path_str = &*path_str;
 
+        let inner_docs = self.cx.tcx.get_attrs(item.def_id).inner_docs();
```

Comment:
Doesn't this also need to check for `is_fake()`? Maybe it would make sense to add an `Item::inner_docs` function to do the check automatically?

Reply:
For some reason I don't understand it wasn't causing any crash, but it could have. I did as you sad and it should be safe now 👍